src/ripple/app/misc/impl/AmendmentTable.cpp

```diff
@@ -402,6 +494,24 @@ AmendmentTableImpl::find(std::string const& name) const
     return {};
 }
 
+void
+AmendmentTableImpl::persistVote(
+    uint256 const& amendment,
+    std::string const& name,
+    bool vote) const
```

Comment:
As with the ctor above, please rename this to `vetoed` or `voteVeto` or whatever.

Reply:
Done.